src/main/updater.ts

```diff
@@ -54,8 +59,8 @@ const intiUpdateHandlers = (browserWindow: BrowserWindow) => {
     if (notifyUserIfNoUpdateAvailable) {
       dialog.showMessageBox(browserWindow, {
         type: 'info',
-        title: 'No update available',
-        message: `No update available`,
+        title: t('No update available'),
```

Comment:
Do we want to follow the same style as the translation keys in other translation files for consistency?

For example, this would be "NoUpdateAvailable"

Reply:
Yeah, we should follow the same format used in other files

Reply:
Alright will do the changes suggested